src/Visualizer.php

```diff
@@ -56,7 +56,7 @@ public function visualize(array $blocks): string
         foreach ($matrix as $name => $row) {
             $lines[] = vsprintf('%s    %s', [
                 str_pad($name, $nameLength, ' '),
-                $this->toBars($row),
```

Comment:
Could you undo these styling changes?

Reply:
Rolled back last commit to undo changes.